ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/location/location.tsx

```diff
@@ -127,6 +131,15 @@ const LocationInput = ({ onChange, value, errorListener }: any) => {
   const [state, setState] = React.useState(locationModel.toJSON() as any)
   const isDrawing = useIsDrawing()
   const { listenTo, stopListening } = useBackbone()
+  const [openColors, setOpenColors] = React.useState(false)
+  const anchorEl = React.useRef(null)
+  const handleOpenColorSelector = () => {
```

Comment:
We should be explicit about the state we want to set. Clicking the color square should set `openColors` to true, not just flip its current value. Likewise, the close handler should set it to false.